manifests/dev/vsphere-67u3/vanilla/vsphere-csi-driver.yaml

```diff
@@ -89,7 +89,7 @@ apiVersion: v1
 data:
   "csi-migration": "false"
   "csi-auth-check": "true"
-  "online-volume-extend": "false"
+  "online-volume-extend": "true" # Kindly use the CSI manifest YAMLs of vSphere 7.0U2 or above to use this feature
```

Comment:
I would say something like this:

`Internal FSS is enabled in v2.2.0 version of the driver. The vSphere need to be at least 7.0 U2 for this feature to work. If the vSphere version is lower, then the online volume expansion will fail with appropriate error message during runtime.`

Reply:
Changed it